backend/danswer/chat/stream_processing/citation_processing.py

```diff
@@ -22,12 +22,16 @@ def __init__(
         self,
         context_docs: list[LlmDoc],
         doc_id_to_rank_map: DocumentIdOrderMapping,
+        display_doc_order_dict: dict[str, int],
         stop_stream: str | None = STOP_STREAM_PAT,
     ):
         self.context_docs = context_docs
         self.doc_id_to_rank_map = doc_id_to_rank_map
         self.stop_stream = stop_stream
         self.order_mapping = doc_id_to_rank_map.order_mapping
```

Comment:
This object would contain both that way we have a single cohesive "state object" we can pass to the function